docs/LONG_LIVED_PERSISTENT_SESSIONS.md

```
@@ -0,0 +1,546 @@
+
+```csharp
+List<ChatTurn> History   // { Role, Content, Timestamp, ToolCalls? }
+```
+
+Each `ChatTurn` carries role (`user`/`assistant`/`system`), content, timestamp, and optional `ToolInvocation` list (tool name, arguments, result, duration, failure codes, etc.).
+
+### 2.3 Lifecycle State
+
+Dual state machines:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)